### PR TITLE
ready for review: update mapping-variant-calling.md

### PR DESCRIPTION
**Pull request by taylorreiter**
PR author:
- [x] pull request is ready for review & merge

Things to check:

- [x] tutorial resets working directory at beginning with a `cd ~/``
- [x] tutorial contains relevant `conda install -y` commands at the beginning
- [ ] tutorial links to previous tutorial (at top) and next tutorial (at bottom)
- [x] tutorial has learning objectives at top

### CODE REVIEW

File: mapping-variant-calling.md [hunk 1]
```diff
@@ -40,43 +40,82 @@ Goal: execute a basic mapping
...
         
 ### Map!
 
+We use an algorithm called `bwa mem` to perform mapping.
+
 ```
 bwa mem -t 4 orf_coding.fasta ERR458493.qc.fq.gz  > ERR458493.sam
 ```
         
+**What is the difference between Salmon and bwa mem?** In an earlier tutorial, we used Salmon "quasi-mapping" to generate
```
Comment (taylorreiter) [inline mapping-variant-calling.md:84]: @AstrobioMike please cross check and change as you see fit :)

Comment (AstrobioMike) [reply mapping-variant-calling.md:84]: perfect!

**Review by AstrobioMike — APPROVED** [review]

**Comment by AstrobioMike** [comment]
I haven't yet run through the code myself, so not merging. But feel free to if someone already has, or i will tomorrow in the morning!

**Comment by AstrobioMike** [comment]
actually i guess it doesn't matter, i'll run through it in the morning too either way :)

**Pull request merged**